### ISSUE TITLE
I got an error when I installed awscrt 

### ISSUE THREAD

**Issue by rokurorock** [awslabs/aws-crt-python#258]
environment
```
(.venv) pi@raspberrypi:~/edge-logger $ uname -a
Linux raspberrypi 5.4.83-v7l+ #1379 SMP Mon Dec 14 13:11:54 GMT 2020 armv7l GNU/Linux
```

python version
```
(.venv) pi@raspberrypi:~/edge-logger $ python --version
Python 3.7.3
```

error detail
```
r(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/pip-wheel-lkd21cmj --python-tag cp37
  usage: -c [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: -c --help [cmd1 cmd2 ...]
     or: -c --help-commands
     or: -c cmd --help

  error: invalid command 'bdist_wheel'
error
  Running setup.py clean for awscrt
  Running command /home/pi/edge-logger/.venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-8dg_yovk/awscrt/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" clean --all
  running clean
  'build/lib.linux-armv7l-3.7' does not exist -- can't clean it
  'build/bdist.linux-armv7l' does not exist -- can't clean it
  'build/scripts-3.7' does not exist -- can't clean it
Failed to build awscrt
Installing collected packages: awscrt
  Created temporary directory: /tmp/pip-record-z9zdia98
  Running setup.py install for awscrt ...     Running command /home/pi/edge-logger/.venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-8dg_yovk/awscrt/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-z9zdia98/install-record.txt --single-version-externally-managed --compile --install-headers /home/pi/edge-logger/.venv/include/site/python3.7/awscrt
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-armv7l-3.7
    creating build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/common.py -> build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/crypto.py -> build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/http.py -> build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/auth.py -> build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/s3.py -> build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/mqtt.py -> build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/exceptions.py -> build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/__init__.py -> build/lib.linux-armv7l-3.7/awscrt
    copying awscrt/io.py -> build/lib.linux-armv7l-3.7/awscrt
    creating build/lib.linux-armv7l-3.7/awscrt/eventstream
    copying awscrt/eventstream/rpc.py -> build/lib.linux-armv7l-3.7/awscrt/eventstream
    copying awscrt/eventstream/__init__.py -> build/lib.linux-armv7l-3.7/awscrt/eventstream
    running build_ext
    --- Building dependency: aws-lc (RelWithDebInfo) ---
    /usr/bin/cmake -DCMAKE_PREFIX_PATH=/tmp/pip-install-8dg_yovk/awscrt/build/deps/install -DCMAKE_INSTALL_PREFIX=/tmp/pip-install-8dg_yovk/awscrt/build/deps/install -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTING=OFF -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_LIBSSL=OFF /tmp/pip-install-8dg_yovk/awscrt/crt/aws-lc
    -- The C compiler identification is GNU 8.3.0
    -- Check for working C compiler: /usr/bin/cc
    -- Check for working C compiler: /usr/bin/cc -- works
    -- Detecting C compiler ABI info
    -- Detecting C compiler ABI info - done
    -- Detecting C compile features
    -- Detecting C compile features - done
    -- Found Perl: /usr/bin/perl (found version "5.28.1")
    -- Checking for module 'libunwind-generic'
    --   Found libunwind-generic, version 1.21
    -- Go not found. Disabling some code generation and using pre-generated code in generated-src/
    -- The ASM compiler identification is GNU
    -- Found assembler: /usr/bin/cc
    CMake Error at CMakeLists.txt:660 (add_subdirectory):
      add_subdirectory given source "util/fipstools/cavp" which is not an
      existing directory.


    CMake Error at CMakeLists.txt:661 (add_subdirectory):
      add_subdirectory given source "util/fipstools/acvp/modulewrapper" which is
      not an existing directory.


    -- Configuring incomplete, errors occurred!
    See also "/tmp/pip-install-8dg_yovk/awscrt/build/deps/aws-lc/CMakeFiles/CMakeOutput.log".
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-8dg_yovk/awscrt/setup.py", line 280, in <module>
        'boto3'
      File "/home/pi/edge-logger/.venv/lib/python3.7/site-packages/setuptools/__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib/python3.7/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/usr/lib/python3.7/distutils/dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/home/pi/edge-logger/.venv/lib/python3.7/site-packages/setuptools/command/install.py", line 61, in run
        return orig.install.run(self)
      File "/usr/lib/python3.7/distutils/command/install.py", line 589, in run
        self.run_command('build')
      File "/usr/lib/python3.7/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/usr/lib/python3.7/distutils/command/build.py", line 135, in run
        self.run_command(cmd_name)
      File "/usr/lib/python3.7/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/tmp/pip-install-8dg_yovk/awscrt/setup.py", line 194, in run
        self._build_dependency(lib)
      File "/tmp/pip-install-8dg_yovk/awscrt/setup.py", line 175, in _build_dependency
        subprocess.check_call(cmake_args)
      File "/usr/lib/python3.7/subprocess.py", line 347, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['/usr/bin/cmake', '-DCMAKE_PREFIX_PATH=/tmp/pip-install-8dg_yovk/awscrt/build/deps/install', '-DCMAKE_INSTALL_PREFIX=/tmp/pip-install-8dg_yovk/awscrt/build/deps/install', '-DBUILD_SHARED_LIBS=OFF', '-DCMAKE_BUILD_TYPE=RelWithDebInfo', '-DBUILD_TESTING=OFF', '-DCMAKE_POSITION_INDEPENDENT_CODE=ON', '-DBUILD_LIBSSL=OFF', '/tmp/pip-install-8dg_yovk/awscrt/crt/aws-lc']' returned non-zero exit status 1.
error
Cleaning up...
  Removing source in /tmp/pip-install-8dg_yovk/awscrt
Removed build tracker '/tmp/pip-req-tracker-sr7jzrib'
Exception information:
Traceback (most recent call last):
  File "/home/pi/edge-logger/.venv/lib/python3.7/site-packages/pip/_internal/cli/base_command.py", line 143, in main
    status = self.run(options, args)
  File "/home/pi/edge-logger/.venv/lib/python3.7/site-packages/pip/_internal/commands/install.py", line 386, in run
    use_user_site=options.use_user_site,
  File "/home/pi/edge-logger/.venv/lib/python3.7/site-packages/pip/_internal/req/__init__.py", line 49, in install_given_reqs
    **kwargs
  File "/home/pi/edge-logger/.venv/lib/python3.7/site-packages/pip/_internal/req/req_install.py", line 791, in install
    spinner=spinner,
  File "/home/pi/edge-logger/.venv/lib/python3.7/site-packages/pip/_internal/utils/misc.py", line 723, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip._internal.exceptions.InstallationError: Command "/home/pi/edge-logger/.venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-8dg_yovk/awscrt/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-z9zdia98/install-record.txt --single-version-externally-managed --compile --install-headers /home/pi/edge-logger/.venv/include/site/python3.7/awscrt" failed with error code 1 in /tmp/pip-install-8dg_yovk/awscrt/
```

**Comment by TingDaoK** [comment]
Hey, we just released v0.11.6, it should the error reported here, can you try again with the latest version?

**Comment by graebm** [comment]
This might be fixed in version [0.11.8](https://github.com/awslabs/aws-crt-python/releases/tag/v0.11.8)
We limited the parallelism during builds
My Raspberry Pi 3B+ was also dying.

**Comment by graebm** [comment]
This is fixed in [v0.11.8](https://github.com/awslabs/aws-crt-python/releases/tag/v0.11.8)
[piwheels.org](https://www.piwheels.org/project/awscrt/) is able to build precompiled wheels for this release, so installation should be quick and seamless
